qutebrowser/config/configdata.yml

```diff
@@ -1201,6 +1201,55 @@ editor.encoding:
   default: utf-8
   desc: Encoding to use for the editor.
 
+## fileselect
+
+fileselect.handler:
+  default: default
+  type:
+    name: String
+    valid_values:
+      - default: "Use the default file selector."
+      - external: "Use an external command."
+  desc: >-
+      Handler for selecting file(s) in forms.
+      If `external`, then the commands specified by
+      `fileselect.single_file.command` and
+      `fileselect.multiple_files.command` are used to select one or
+      multiple files respectively.
+
+fileselect.single_file.command:
+  type:
+    name: ShellCommand
+    placeholder: true
+    completions:
+      - ['["xterm", "-e", "ranger --choosefile={}"]', "Ranger in xterm"]
+      - ['["xterm", "-e", "vifm", "--choose-file", "{}"]', "vifm in xterm"]
```

Comment:
FYI, looks like `vifm --chose-file` doesn't actually exist... Changed to `--choose-files` in 8faa60592.

Reply:
Indeed seems like `vifm --choose-file` silently fallbacks to `vifm --choose-files`.